storage/rocksdb/rdb_datadic.cc

```diff
@@ -2692,6 +2692,20 @@ void Rdb_ddl_manager::erase_index_num(const GL_INDEX_ID &gl_index_id) {
   m_index_num_to_keydef.erase(gl_index_id);
```

Comment:
why aren't we locking here? std::map isn't thread safe, and couldn't this be accessed from multiple threads?